Library/Formula/glib.rb

```diff
@@ -3,6 +3,7 @@ class Glib < Formula
   homepage "https://developer.gnome.org/glib/"
   url "https://download.gnome.org/sources/glib/2.46/glib-2.46.1.tar.xz"
   sha256 "5a1f03b952ebc3a7e9f612b8724f70898183e31503db329b4f15d07163c8fdfb"
+  revision 1
```

Comment:
I'd rather the people who need the static libraries reinstall `glib` via `brew` themselves rather than making every single Homebrew user who has `glib` installed rebuild personally, unless this is likely to hit a particularly vast number of people.